app/assets/javascripts/map/components/registration-form.js

```diff
@@ -3,6 +3,10 @@
 
 function RegistrationForm() {
   let alert = null // { type: 'error', message: "This is a test of the alert message." }
+
+  const regexForEmailValidation = RegExp(/^\w+([\.-]?\w+)@\w+([\.-]?\w+)(\.\w{2,3})+$/);
+  let emailErrorMessage = null;
```

Comment:
Instead of creating a new variable, I think it might be better to reuse our alert message above. If you set
```javascript
alert = { type: 'error', message: Util.translate('registration.form.emailInvalidErrorMessage') }
```
Then that should make an error message appear above the "Submit" button. That would probably also be good, because sometimes the form can get longer, and the user _might_ not be able to see the email field when they are submitting.